services/gitea/gitea-helper.js

```diff
@@ -0,0 +1,13 @@
+const documentation = `
+Note that only internet-accessible Gitea compatible instances are supported, for example
+[https://codeberg.org](https://codeberg.org).
+`
+
+function httpErrorsFor(notFoundMessage = 'user or repo not found') {
+  return {
+    401: notFoundMessage,
```

Comment:
I think on GitLab we had to classify 401 as a "not found" because of the way GitLab's API handles private resources. Not sure that transfers here though. When would this API return a 401?

Reply:
403 is returned by the API, have updated appropriately.